src/WebdriverClassicDriver.php

```diff
@@ -18,14 +18,18 @@
 use Facebook\WebDriver\Exception\TimeoutException;
 use Facebook\WebDriver\Exception\UnsupportedOperationException;
 use Facebook\WebDriver\Exception\WebDriverException;
+use Facebook\WebDriver\Interactions\WebDriverActions;
+use Facebook\WebDriver\Internal\WebDriverLocatable;
```

Comment:
The `Internal` namespace suggests, that this class/interface shouldn't be used outside of the library and therefore not included in BC promise. But maybe I'm mistaken.